src/test_reference.jl

```diff
@@ -117,6 +117,13 @@ function test_reference(
     rendermode=nothing;
     kw...) where {F <: DataFormat, T}
 
+    if haskey(kw, :return_bool)
+        return_bool = kw[:return_bool]::Bool
+    else
+        return_bool = false
+    end
+    kw = filter(p -> p[1] != :return_bool, kw)
```

Comment:
Just a suggestion for simpler code
```suggestion
    return_bool  = get(kw, :return_bool, false)::Bool
    delete!(kw, :return_bool)
```

Reply:
The `return_bool = get(kw, :return_bool, false)::Bool` part is good.

The `delete!` throws a `MethodError`.